nodejs/tests/api.key.test.js

```diff
@@ -0,0 +1,33 @@
+require('dotenv').config();
+console.clear();
+
+const axios = require('axios');
+const urlHandler = require('../handlers/url.handler');
+
+
+// Arkhia config
+const apiKey = process.env.ARKHIA_API_KEY;
+const headers = { 'x-api-key': apiKey };
+
+describe('Test to validate API key', () => {
+    test('Should load correct API Key', () => {
+        expect(apiKey).toBeDefined();
+        expect(apiKey.length).toEqual(32);
+    });
+
+    test('this API Key is not valid', async () => {
+        try {
+            const balancesUrl = `${urlHandler.getApiUrl()}/balances`;
+            const response = await axios.get(balancesUrl, { headers });
+        } catch (error) {
+            expect((error.response.data)).toHaveProperty('status', false);
+            expect(error.response.data.response).toContain('Unauthorized');
+            console.error(error.response.data.response)
+        }
+
+
+    });
+
+
```

Comment:
Missing 2

1.
Pricing tier > API Key free testing a paid tier
> API Key paid testing a free tier

2. If no api key should return adequate payload